src/database/mod.rs

```diff
@@ -401,18 +401,18 @@ impl KeyValueDatabase {
 		let sqlite_exists = path.join("conduit.db").exists();
 		let rocksdb_exists = path.join("IDENTITY").exists();
 
-		let mut count = 0;
+		let mut count: u8 = 0;
 
 		if sqlite_exists {
-			count += 1;
+			count = count.saturating_add(1);
 		}
 
 		if rocksdb_exists {
-			count += 1;
+			count = count.saturating_add(1);
 		}
 
 		if count > 1 {
```

Comment:
This is a bit silly honestly, I don't think we're planning on adding *more* database backends, just use `sqlite_exists && rocksdb_exists`

Reply:
Removed the counter stuff